content/docs/observable.md

```diff
@@ -0,0 +1,202 @@
+# Observable
+
+> [tip box yellow] This functionality was added in v2.0.0-alpha.202 and is still experimental.
+
+### TLDR
+Imba provides MobX-like reactivity implemented as decorators.
+- Mark class and tag members as observable with the `@observable` decorator.
+- Methods decorated with `@autorun` will be reactively invoked whenever
+referenced observables change.
+- Getters decorated with `@computed` will be cached on first use
+and only recomputed whenever referenced observables change.
+
+See also [additional details](#additional-details).
+
+### Guide
+In Imba, it's important to keep in mind that [commit](/api/imba/commit) is called after every handled event.
+This means that state changes which occur inside of event handlers will be reflected in the next render.
+We can apply this knowledge to implement a simplistic search:
```

Comment:
The description says "every handled event" but the example doesn't have any events or event handlers (it uses bind) which could be confusing since bind kinda does the events behind the scenes. Maybe we can make this explicit with an actual event handler that updates query?